nestkernel/archiving_node.cpp

```diff
@@ -99,7 +103,7 @@ nest::Archiving_Node::get_K_value( double t )
     if ( t > history_[ i ].t_ )
     {
       return ( history_[ i ].Kminus_
-        * std::exp( ( history_[ i ].t_ - t ) / tau_minus_ ) );
+        * std::exp( ( history_[ i ].t_ - t ) * tau_minus_inv_ ) );
```

Comment:
I'd be curious if there's a way to state this in a more numerically stable way -- just to keep in mind for the future